src/main/content/index.html

```diff
@@ -106,6 +106,15 @@ <h1 class="white-text expressive-heading-06">{{t.index.welcome-kabanero}}</h1>
                 </li>
             </ul>
 
+            <p>{{t.index.devtools.steps.codeready}}</p>
+            <ul>
+                <li>
+                    {{t.index.devtools.steps.install}}
+                    <a target="_blank"
+                        href="/docs/ref/general/installation/installing-codeready-and-codewind">{{t.index.devtools.steps.codereadyInstall}}</a>
+                </li>
```

Comment:
I think we should take it 1 step further and link to codeready if its installed already. So same thing as what we already do for kAppNav in `home.js` 

might have to change the text to match as well, for example if it says install codeready (but links to codeready) that wouldn't make sense. Feel free to think of a generalized string too, one that would make sense in both occasions (not installed- point to docs & installed - point to codeready app) to reduce the complexity.